demo/admin/src/products/ProductForm.tsx

```diff
@@ -177,6 +179,17 @@ export function ProductForm({ id }: FormProps) {
                         <TextField required fullWidth name="title" label={<FormattedMessage id="product.title" defaultMessage="Title" />} />
                         <TextField required fullWidth name="slug" label={<FormattedMessage id="product.slug" defaultMessage="Slug" />} />
 
+                        <Field
+                            name="priceRange"
+                            label={<FormattedMessage id="product.priceRange" defaultMessage="Price range" />}
+                            fullWidth
+                            required
+                            component={FinalFormRangeInput}
+                            min={5}
+                            max={100}
+                            startAdornment={<span>€</span>}
```

Comment:
Please wrap the symbol in an `InputAdornment`, as defined by MUI: https://mui.com/material-ui/react-text-field/#input-adornments

```suggestion
                            startAdornment={<InputAdornment position="start">€</InputAdornment>}
```